server/sonar-web/src/main/js/apps/marketplace/components/PluginActions.tsx

```diff
@@ -69,47 +69,53 @@ export default class PluginActions extends React.PureComponent<Props, State> {
   handleUninstall = () => this.doPluginAction(uninstallPlugin);
   handleTermsCheck = (checked: boolean) => this.setState({ acceptTerms: checked });
 
+  renderBundled() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The function render has a cyclomatic complexity of 15 which is higher than the threshold of 10 [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1541)